server/src/test/java/io/spine/examples/kanban/server/board/BoardInitProcessTest.java

```diff
@@ -52,48 +53,86 @@ void sendCommand() {
-        int columnCount = defaultColumnCount();
-        for (int i = 0; i < columnCount; i++) {
-            commands.message(i)
-                    .comparingExpectedFieldsOnly()
-                    .isEqualTo(commandForBoardInit);
+    void issuesCommands() {
+        CommandSubject issuedCommands = context().assertCommands()
+                                                 .withType(CreateColumn.class);
+        int expectedCount = DefaultColumns.count();
+        issuedCommands.hasSize(expectedCount);
+
+        List<CreateColumn> expectedCommands = expectedCommands();
+        for (int i = 0; i < expectedCount; i++) {
+            issuedCommands.message(i)
+                          .comparingExpectedFieldsOnly()
+                          .isEqualTo(expectedCommands.get(i));
         }
     }
 
+    private List<CreateColumn> expectedCommands() {
+        List<CreateColumn> commands = new ArrayList<>();
+
+        for (BoardInit.DefaultColumn c : DefaultColumns.all()) {
+            commands.add(
+                    CreateColumn.newBuilder()
+                                .setBoard(board())
+                                .setBoardInit(true)
+                                .setName(c.name())
+                                .buildPartial()
+            );
+        }
+
+        return commands;
+    }
+
     @Test
     @DisplayName("create default columns")
     void createsColumns() {
-        Collection<ColumnId> defaultColumns = createdColumns();
-        defaultColumns.forEach(
-                c -> context().assertEntityWithState(c, Column.class)
-                              .exists()
+        List<Column> expectedColumns = expectedColumns();
+        expectedColumns.forEach(
+                c -> context().assertEntityWithState(c.getId(), Column.class)
+                              .hasStateThat()
+                              .isEqualTo(c)
         );
     }
 
+    private List<Column> expectedColumns() {
+        return context().assertCommands()
+                        .withType(CreateColumn.class)
+                        .actual()
+                        .stream()
+                        .map(c -> {
+                            CreateColumn command = unpack(c.getMessage(), CreateColumn.class);
+
+                            return Column.newBuilder()
+                                        .setId(command.getColumn())
+                                        .setBoard(command.getBoard())
+                                        .setName(command.getName())
+                                        .vBuild();
+                        })
+                        .collect(toList());
+    }
+
     @Test
-    @DisplayName("delete itself when finished")
-    void isDeletedWhenFinished() {
+    @DisplayName("emit the `BoardInitialized` event when terminated")
+    void emitsEvent() {
+        EventSubject events = context().assertEvents()
+                                       .withType(BoardInitialized.class);
+
```

Comment:
Please have less (!) empty lines. 

I wouldn't have any empty lines in this method, as there is more than enough spacing already.